### PR TITLE
MP-758 Updated hooks and removed them from the path

### PR DESCRIPTION
**Pull request by nessy**

### CODE REVIEW

**Comment by Shodske** [comment]
Closing for now as it is still work in progress.

File: specification/paths/Webhooks-webhook_id.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
   "get": {
     "tags": [
-      "Webhooks"
+      "⚠ Webhooks"
```
Comment (nessy) [inline specification/paths/Webhooks-webhook_id.json]: Should hooks not be removed completely?

File: specification/paths/Webhooks.json [hunk 2]
```diff
@@ -1,7 +1,7 @@
 {
   "get": {
     "tags": [
-      "Webhooks"
+      "⚠ Webhooks"
```
Comment (nessy) [inline specification/paths/Webhooks.json]: Should hooks not be removed completely?

File: specification/paths/Webhooks-webhook_id.json (hunk 1 above)
Comment (Shodske) [reply specification/paths/Webhooks-webhook_id.json]: Removing them completely right now will cause the API tests to fail.

**Pull request merged**